pkg/volume/glusterfs/glusterfs.go

```diff
@@ -490,13 +495,15 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 	if len(pv.Spec.AccessModes) == 0 {
 		pv.Spec.AccessModes = r.plugin.GetAccessModes()
 	}
+	sGid := strconv.FormatInt(reqGid, 10)
+	pv.Annotations = map[string]string{"pv.beta.kubernetes.io/gid": sGid}
```

Comment:
@liggitt but we can use it for alpha? (trying to read back the history which is quite lengthy) Could we introduce this field as an alpha to see if we want to continue down this route? Can you mix these?